src/ape/managers/accounts.py

```diff
@@ -317,3 +336,12 @@ def __contains__(self, address: AddressType) -> bool:
             any(address in container for container in self.containers.values())
             or address in self.test_accounts
         )
+
+    def use_sender(self, account, passphrase: Optional[str] = None):
+        if not isinstance(account, AccountAPI):
+            try:
+                account = self.load(account)  # if account is alias
+            except (ValueError, IndexError):
+                account = self[account]
+        account.set_autosign(True, passphrase)
```

Comment:
I don't think autosign should automatically be enabled behind the user's back.
We could have a kwarg `autosign: bool` though.
Otherwise, it makes this method a lot more dangerous if the user is not aware of this.

Reply:
Note: this is the main blocker for me right now!